src/generate/templates/header.rs

```diff
@@ -143,8 +143,8 @@ pub enum ListHead<C> {
     Nil,
 }
 
-impl<'a, 'i, I: ::gll::runtime::Input, T> Handle<'a, 'i, I, [T]> {
-    fn one_list_head(self) -> ListHead<Result<(Handle<'a, 'i, I, T>, Handle<'a, 'i, I, [T]>), Ambiguity<Self>>> {
+impl<'a, 'i, I: crate::gll::runtime::Input, T> Handle<'a, 'i, I, [T]> {
+    fn one_list_head(self) -> ListHead<Result<(Handle<'a, 'i, I, T>, Self), Ambiguity<Self>>> {
```

Comment:
Heh, there's a similar one... somewhere. I think in an `-> impl Iterator`? Which used to be broken?
Maybe we changed that one and not this one.